response_operations_ui/controllers/notify_controller.py

```diff
@@ -39,16 +39,19 @@ def _send_message(self, email, template_id, personalisation=None, reference=None
             notification.update({"reference": reference})
 
         url = f'{self.notify_url}{template_id}'
-
         auth = app.config['SECURITY_USER_NAME'], app.config['SECURITY_USER_PASSWORD']
         response = requests.post(url, auth=auth, json=notification)
-        if response.status_code == 201:
+        status_code = response.status_code
+
+        try:
+            response.raise_for_status()
             logger.info('Notification id sent via Notify-Gateway to GOV.UK Notify.', id=response.json().get("id"))
-        else:
+        except HTTPError:
             ref = reference if reference else 'reference_unknown'
-            raise NotifyError("There was a problem sending a notification to Notify-Gateway to GOV.UK Notify",
-                              reference=ref)
-
+            logger.error('There was a problem sending a notification to Notify-Gateway', 
+                         ref=ref, status_code=status_code, message=response.text)
```

Comment:
It's minor, but we only use that status_code variable once, and only if there's an error.  Might as well just do `status_code=response.status_code` and do away with assigning the variable above